Module.yaml

```diff
@@ -0,0 +1,91 @@
+version: 1.0.0
+name: Melita IO Batch for devices controller
+description: Module for communication with melita.io API for devices controller, for sending requests to multiple devices
+icon: https://icons-020-demo.s3.eu-central-1.amazonaws.com/mi_melita_output.png
+type: process
+tags:
+  - Melita.io
+categories:
+  - id: category
+    name: Ouput
+image:
+  name: weevenetwork/melita-io-batch
+  tag: latest
+environments:
+  - key: INGRESS_HOST
+    description: Host name/address where the server will run.
+    value: ''
+    default: 0.0.0.0
+    type: string
+    name: Host Name
+  - key: INGRESS_PORT
+    description: Port on which the server will run.
+    value: ''
+    default: 80
+    type: integer
+    name: Port
```

Comment:
If it is process/output module then we don't need these. It's set by weeve Agent.